pkgs/development/python-modules/panflute/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, python3Packages
```

Comment:
This is now unused

Reply:
and python3Packages can't be used in python-modules.